app/db/migrations/20201223200005-default-user-role.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
```

Comment:
Interesting. Could you describe what some of the pain points are and what `umzug` could do to resolve those? (I'm not familiar with this tool.)

Reply:
So coming from Django, Django is nice because you rarely need to write migrations, you can have them auto generated. umzug appears to be an attempt to have similar functionality for sequelize